### DISCUSSION TITLE
PLZ HELP... My max image generation sizes keep getting significantly smaller and smaller

### DISCUSSION THREAD

**Discussion by ArchAngelAries** [lshqqytiger/stable-diffusion-webui-directml#175] [Q&A] [answered]
I'm on Windows 11 and AMD 7900 XT. A couple months ago I could generate just about anything after at a total max of 1024 x 1536. Now I can only run a total max of 720 x 864. Does having more/certain extensions or number of installed models affect the VRAM? Why is it with every new commit my ability to generate gets whittled down further and further? I've tried every combination of COMMANDLINE_ARGS I can find, I've added --backend=directml , I've reinstalled everything countless times, I tried deleting some stuff from the ARGS and changing some of the new optimization settings, but they just caused a bunch of errors. I'm losing my mind. At this point I'm about to give up... Getting super bummed out and could really use some help....

My current webui-user.bat:

set PYTHON=
set GIT=
set VENV_DIR=
set PYTORCH_CUDA_ALLOC_CONF=garbage_collection_threshold:0.9,max_split_size_mb:512
set COMMANDLINE_ARGS=--backend=directml --medvram --precision full --no-half --no-half-vae --opt-split-attention --disable-nan-check --autolaunch --api
set SAFETENSORS_FAST_GPU=1

call webui.bat

**Comment by xil64** [comment, marked as answer]
I have a 7900XT as well, and have had this same problem.  I spent a long time tweaking settings, and my best results are with these commandline_args:

set COMMANDLINE_ARGS=--medvram --upcast-sampling --no-half-vae --disable-nan-check  --use-cpu interrogate --opt-split-attention-v1

Also check under Settings/Optimizations, cross attention optimization is set to V1.  This setting seems to overwrite the commandline arg.  I've also had good luck setting Token Merging Ratio to 0.5 to speed up generations a lot (which seems to produce only NANs for me with sub-quad attention)

with these settings and a SD 1.5 model I can do batches of 4 512x768 hires.fix to x2 (1024x1536 final res).

**Reply by ArchAngelAries** [reply]
Using your settings and V1 I wasn't able to generate anything with highres fix, oddly enough though, using Doggettx I'm able to generate a batch of 2 at 1024x1536 final res after high res fix, ranging an average 0f 1.2 - 1.8s/it. Was wondering if you have any other settings in your webui-user.bat... like "set PYTORCH_CUDA_ALLOC_CONF=garbage_collection_threshold:0.9,max_split_size_mb:512" or "set SAFETENSORS_FAST_GPU=1"?

**Reply by xil64** [reply]
I've tried with and without the PYTORCH_CUDA_ALLOC_CONF line and I'm not sure if it does anything or not.  I am leaning towards it doing nothing.

I did forget to mention to use Tiled VAE extension.  I set Encoder/Decoder at 1024/128, without this I can't generate anything either.

my pull is also about 2 weeks old, I noticed some big changes were just pushed today so if you're using that pull, YMMV.
